Reconstruction/RecCalorimeter/src/components/CreateCaloCells.cpp

```diff
@@ -184,7 +165,7 @@ StatusCode CreateCaloCells::prepareEmptyCells(std::vector<fcc::CaloHit*>& caloCe
   // Get the total number of active volumes in the geometry
   auto highestVol = gGeoManager->GetTopVolume();
   // Substract volumes with active material which are not readout (e.g. ECAL: bath volume)
-  unsigned int numLayers = det::utils::countPlacedVolumes(highestVol, m_activeVolumeName)-m_numVolumesRemove;
+  unsigned int numLayers = det::utils::countPlacedVolumes(highestVol, m_activeVolumeName);
```

Comment:
Doesn't that change invalidate the comment just above?
And for my understanding - you could drop the subtraction because you now start with active volumes in the first place?